src/controller/CommissioningDelegate.h

```diff
@@ -507,6 +510,41 @@ class CommissioningParameters
         return *this;
     }
 
+    Optional<NodeId> GetICDControllerNodeId() const { return mICDControllerNodeId; }
+    CommissioningParameters & SetICDControllerNodeId(NodeId icdControllerNodeId)
+    {
+        mICDControllerNodeId = MakeOptional(icdControllerNodeId);
+        return *this;
+    }
+
+    Optional<uint64_t> GetICDSubjectId() const { return mICDSubjectId; }
```

Comment:
And this should be MonitoredSubject, again to match spec.

Reply:
done